erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -70,6 +70,12 @@ def on_submit(self):
 		self.update_advance_paid()
 		self.update_expense_claim()
 
+	def before_print(self):
+		self.gl = frappe.get_list("GL Entry",filters={"voucher_type": "Payment Entry",
```

Comment:
No one can understand the meaning of gl, make it gl_entries (self.gl_entries)

Reply:
done